muikku/src/main/webapp/WEB-INF/templates/flex-base.xhtml

```diff
@@ -20,6 +20,7 @@
           var MUIKKU_LOGGED_USER_ID = parseInt('#{sessionBackingBean.loggedUserId}')||null;
           var MUIKKU_LOGGED_USER = '#{sessionBackingBean.loggedUser}'||null;
           var MUIKKU_LOGGEDIN = #{sessionBackingBean.loggedIn};
+          var CHAT_CHECKBOX_PERMISSION=#{sessionBackingBean.hasEnvironmentPermission('CHAT_CHECKBOX_PERMISSION')};
```

Comment:
Is there any better place for this variable? Like a chat.xhtml included in the page or such? Would make more sense in there rather than in this very generalized place.